module/os/operation_siren.py

```diff
@@ -221,6 +247,10 @@ def clear_os_world(self):
         # Force to use AP boxes
         backup = self.config.cover(OS_ACTION_POINT_PRESERVE=40, OS_ACTION_POINT_BOX_USE=True)
 
+        # Before start, repair if any ship is below
+        # a fixed threshold of 85% as precaution
+        self.handle_fleet_repair(True, 0.85, revert=False)
+
```

Comment:
Call `self.handle_fleet_repair` and use the default settings would be enough.

Reply:
Understood, it will just use configured settings. No need for precaution.